plugins/wporg-5ftf/includes/pledge.php

```diff
@@ -127,5 +128,48 @@ function create_new_pledge( $name ) {
 		'post_status' => 'draft',
 	);
 
-	return wp_insert_post( $args, true );
+
+	$pledge_id = wp_insert_post( $args, true );
+	// The pledge's meta data is saved at this point via `save_pledge_meta()`, which is a `save_post` callback.
+
+	if ( ! is_wp_error( $pledge_id ) ) {
+		send_pledge_verification_email( $pledge_id, get_post()->ID );
+		send_contributor_verification_email();
```

Comment:
Doh, great catch. I'll remove that since it's just a stub anyway, and then add it in the correct place when I start on #12.

I'll also update the strings to reflect that the contributor emails aren't sent until after they've confirmed the pledge address.